### PR TITLE
Build CompanionPane with Jetpack Compose

### PR DESCRIPTION
**Pull request by joyl1216**

### CODE REVIEW

**Comment by joyl1216** [comment]
Screenshot_20200924-160955

File: DuoComposeSampleApps/ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package com.microsoft.device.display.samples.listdetail
```
Comment (CesarValiente) [inline DuoComposeSampleApps/ListDetail/src/androidTest/java/com/microsoft/device/display/samples/listdetail/ExampleInstrumentedTest.kt:1]: I think you removed the dummy test from other composable samples, so we can delete this as well.

File: DuoComposeSampleApps/ListDetail/src/test/java/com/microsoft/device/display/samples/listdetail/ExampleUnitTest.kt [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```
Comment (CesarValiente) [inline DuoComposeSampleApps/ListDetail/src/test/java/com/microsoft/device/display/samples/listdetail/ExampleUnitTest.kt:12]: Same here, we can remove tests that don't add any value.

File: DuoComposeSampleApps/extendedcanvas/src/test/java/com/microsoft/device/display/samples/extendedcanvas/ExampleUnitTest.kt [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```
Comment (CesarValiente) [inline DuoComposeSampleApps/extendedcanvas/src/test/java/com/microsoft/device/display/samples/extendedcanvas/ExampleUnitTest.kt:12]: Same here, we can delete dummy tests.

File: DuoComposeSampleApps/twopage/src/androidTest/java/com/microsoft/device/display/samples/twopage/ExampleInstrumentedTest.kt [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```
Comment (CesarValiente) [inline DuoComposeSampleApps/twopage/src/androidTest/java/com/microsoft/device/display/samples/twopage/ExampleInstrumentedTest.kt:17]: Same here, we can delete all dummy tests.

File: DuoComposeSampleApps/twopage/src/test/java/com/microsoft/device/display/samples/twopage/ExampleUnitTest.kt [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```
Comment (CesarValiente) [inline DuoComposeSampleApps/twopage/src/test/java/com/microsoft/device/display/samples/twopage/ExampleUnitTest.kt:12]: Same here, we can delete all dummy tests.

File: DuoComposeSampleApps/dualview/src/androidTest/java/com/microsoft/device/display/samples/dualview/ExampleInstrumentedTest.kt [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```
Comment (CesarValiente) [inline DuoComposeSampleApps/dualview/src/androidTest/java/com/microsoft/device/display/samples/dualview/ExampleInstrumentedTest.kt:17]: Same here, we can delete all dummy tests.

File: DuoComposeSampleApps/dualview/src/test/java/com/microsoft/device/display/samples/dualview/ExampleUnitTest.kt [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+import org.junit.Test
+
+import org.junit.Assert.*
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+class ExampleUnitTest {
```
Comment (CesarValiente) [inline DuoComposeSampleApps/dualview/src/test/java/com/microsoft/device/display/samples/dualview/ExampleUnitTest.kt:12]: Same here, we can delete all dummy tests.

File: DuoComposeSampleApps/extendedcanvas/src/androidTest/java/com/microsoft/device/display/samples/extendedcanvas/ExampleInstrumentedTest.kt [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```
Comment (CesarValiente) [inline DuoComposeSampleApps/extendedcanvas/src/androidTest/java/com/microsoft/device/display/samples/extendedcanvas/ExampleInstrumentedTest.kt:17]: Same here, we can delete all dummy tests.

**Review by CesarValiente — APPROVED** [review]
LGTM!  great work Joy!! Compose pro! 🚀

Since you removed dummy tests from some samples, you can delete them all in the different samples added in this PR.

**Comment by joyl1216** [comment]
Update the sample with the new design.
Screenshot_20210108-112521
Screenshot_20210108-112554-1
Screenshot_20210108-112534-2
Screenshot_20210108-112544-3

**Comment by CesarValiente** [comment]
I'm a bit lost with this PR Joy, was this content not already merged before? (`168` files changed)

**Comment by joyl1216** [comment]
@CesarValiente Unfortunately no, we paused it since I was told that I need to rebuild it with the new design. Now it is completed with the new design.

**Pull request closed without merging**